mbed_lstools/lstools_base.py

```diff
@@ -73,6 +73,8 @@ def __init__(self):
         self.lock_file = os.path.join(MbedLsToolsBase.HOME_DIR,
                                       MbedLsToolsBase.MBEDLS_HOME_DIR, MbedLsToolsBase.MBEDLS_GLOBAL_LOCK)
         self.mbedls_get_mocks()
+        # Make sure retargeting is applied if retarget file exists
+        self.retarget(True)
```

Comment:
@mazimkhan Are you sure? That way, skip_retarget will stop working (as you correctly pointed out)